src/webportal/src/assets/img/distribute-hover.svg

```diff
@@ -0,0 +1 @@
+<svg id="layer_1" data-name="layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 75 71"><defs><style>.cls-1,.cls-2,.cls-3{fill:none;}.cls-2,.cls-3{stroke:#0070bc;stroke-width:4px;}.cls-3{stroke-linejoin:round;}</style></defs><title>icons</title><g id="组_13595" data-name="组 13595"><g id="矩形_27935-2" data-name="矩形 27935-2"><rect class="cls-1" x="53" width="22" height="22"/><rect class="cls-2" x="55" y="2" width="18" height="18"/></g><g id="矩形_27937-2" data-name="矩形 27937-2"><rect class="cls-1" x="53" y="49" width="22" height="22"/><rect class="cls-2" x="55" y="51" width="18" height="18"/></g><path id="路径_9288-2" data-name="路径 9288-2" class="cls-3" d="M54.58,11,21.1,34.91,54.58,58.6"/><g id="矩形_27939-2" data-name="矩形 27939-2"><rect class="cls-1" y="24" width="22" height="22"/><rect class="cls-2" x="2" y="26" width="18" height="18"/></g></g></svg>
```

Comment:
there're multiple Chinese id in `svg` files, and I think you can format these files to multiple lines later?

Reply:
done